assets/js/blocks/catalog-sorting/style.scss

```diff
@@ -0,0 +1,8 @@
+.wc-block-catalog-sorting {
+	display: flex;
+
+	.woocommerce-ordering,
+	> div {
+		margin-left: auto;
+	}
+}
```

Comment:
I think we don't need to push the block to the right by default.

I can see that we're inheriting the alignment from the default Woo archive template. But again from that default, we have Product Results Count and Catalog Sorting displayed in the same row, to achieve that, we still need to use a Row block to wrap those two.

On the other hand, personally, aligning the block to the right makes me think wonder if I can change it to the left, but we can't do that.

Reply:
Makes sense, will remove the syling.